t/02-rakudo/03-corekeys.t

```diff
@@ -813,6 +813,7 @@ my @allowed =
         Q{&postcircumfix:<[; ]>},
         Q{&postcircumfix:<{; }>},
         Q{&prefix:<//>},
+        Q{&rmdir},
```

Comment:
I'm not sure why this test only requires the addition of `&rmdir` and not the other subs I adjusted